@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -854,7 +854,8 @@ const createNewVM = async () => {
       throw new Error('Template UUID and Pool ID are required')
     }
 
-    await createVM(vmData.value, vmState.pool.id)
+    const createVmJob = useVmCreateJob(vmData.value, vmState.pool!.id)
+    await createVmJob.run()
```

Comment:
I am not sure to understand how `defineJob` work, but why you never use `createVmJob.validate()`?